config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/environment/EnvironmentBundleCreator.java

```diff
@@ -69,23 +71,22 @@ public Bundle createEnvironmentBundle(Map<String, String> environmentProperties,
         final DocumentBuilder documentBuilder = documentTools.getDocumentBuilder();
         final Document document = documentBuilder.newDocument();
 
+        // Passing Bundle name and version string with config env name (<name>-<version>-*env.install.bundle) as project name
         Map<String, BundleArtifacts> bundleElements = bundleEntityBuilder.build(environmentBundle,
-                EntityBuilder.BundleType.ENVIRONMENT, document, bundleFileName, "", null);
-        for (Map.Entry<String, BundleArtifacts> entry : bundleElements.entrySet()) {
-            String bundleName = entry.getKey();
-            int index = bundleName.lastIndexOf(BUNDLE_EXTENSION);
-            documentFileUtils.createFile(entry.getValue().getBundle(), new File(bundleFolderPath,
-                    bundleName).toPath());
-            documentFileUtils.createFile(entry.getValue().getDeleteBundle(), new File(bundleFolderPath,
-                    bundleName.substring(0, index)+ DELETE_BUNDLE_EXTENSION).toPath());
+                EntityBuilder.BundleType.ENVIRONMENT, document, new ProjectInfo(envInstallBundleFilename, EMPTY, EMPTY));
```

Comment:
its better not to change project name in ProjectInfo, it makes code less readable and later if some code requires actual project name its difficult to get it

Reply:
@uppoju This is how we were doing it before, I just replaced separate variables into 1 object. I wanted to refactor this but it needs a lot of changes at other places as well so skipping it for now.

Reply:
OK